operations/template/alert.tf

```diff
@@ -61,21 +81,40 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_a
     threshold = 1
   }
 
-  #   below tags are managed by CDC
   lifecycle {
     ignore_changes = [
-      tags["business_steward"],
-      tags["center"],
-      tags["environment"],
-      tags["escid"],
-      tags["funding_source"],
-      tags["pii_data"],
-      tags["security_compliance"],
-      tags["security_steward"],
-      tags["support_group"],
-      tags["system"],
-      tags["technical_steward"],
-      tags["zone"]
+      # Ignore changes to tags because the CDC sets these automagically
+      tags,
```

Comment:
The reason the tags were enumerated separately was because we only wanted to ignore the automatically added tags from the CDC, not any tags added by us if we ever were to.

Reply:
See: https://github.com/CDCgov/trusted-intermediary/issues/1396#issuecomment-2429878961

We reverted back to the previous style of `ignore_changes`